### PR TITLE
NN-4109 - restyle UI controls on visitor page

### PR DESCRIPTION
**Pull request by HamHazJustice**

### CODE REVIEW

**Comment by timoj58** [comment]
this branch needs to have the latest main changes merged into it

File: views/prisonerProfile/prisonerVisits/partials/prisonerVisitsSearchForm.njk [hunk 1]
```diff
@@ -15,7 +15,7 @@
     <div class="horizontal-form govuk-!-margin-bottom-3">
       {{ datePicker({
         id: 'fromDate',
-        label: 'Date from',
+        label: 'From',
```
Comment (natclamp-moj) [inline views/prisonerProfile/prisonerVisits/partials/prisonerVisitsSearchForm.njk:18]: Is changing this label critical to correcting the styling? We shouldn't really change things like this in the design without a really good reason/if a designer says it needs doing

Comment (hamidhazrati) [reply views/prisonerProfile/prisonerVisits/partials/prisonerVisitsSearchForm.njk:18]: For the sake of consistency with other date from/to labels, then I think we can.
This label appears 6/7 times in 3 other 'njk' files
Perhaps we can create a ticket in backlog to address those to?
@SimonMitchellMOJ  any thoughts ?

Comment (HamHazJustice) [reply views/prisonerProfile/prisonerVisits/partials/prisonerVisitsSearchForm.njk]: SimonC agrees with you , so reverting those back.

**Review by natclamp-moj — APPROVED** [review]

**Pull request merged**